Message/test/new_embed.js

```diff
@@ -0,0 +1,229 @@
+/**
+ * @fileOverview tests.js Gives testing code for the new page.
+ * This spec is managed by the Jasmine testing library.
+ **/
+
+/*global __html__, describe, beforeEach, afterEach, spyOn, jasmine, it, expect */
+/*global chrome:true, mockValues:true, chromeSentMessages: true */
+/*global privlyNetworkService, loginCheckingCallback, attachEventHandlers, resetGlobals */
+
+describe("Embed Posting Application", function () {
+
+  // Get an HTML document defined by the pre-processor.
+  // This is a rough hack because HTML2JS seems to assign the
+  // key to the absolute URL, which is not reliable on
+  // continuous integration.
+  beforeEach(function () {
+    var keys = Object.keys(__html__);
+    var selectKey;
+    keys.forEach(function (key) {
+      if (key.indexOf("Message/new_embed.html") >= 0) {
+        selectKey = key;
+      }
+    });
+    document.body.innerHTML = __html__[selectKey];
```

Comment:
We are in the process of deprecating the use of the HTML2JS hack. See [Headless Browsers and Fixtures](https://github.com/privly/privly-applications/blob/master/test/README.md)